### PR TITLE
Update deps and fix tests

### PR DESCRIPTION
**Pull request by hacdias**

### CODE REVIEW

**Comment by hacdias** [comment]
Adding `err.notFound = true` to line 74 seems to solve the problem. I think it wants us to specify that key when the error is Not Found.

**Comment by hacdias** [comment]
@diasdavid I just did a [small workaround](https://github.com/ipfs-shipyard/ipfs-blob-store/pull/20/commits/e9f4a907f839c80cfb17d202b84ffed30ba3d507) to fix this. If you agree, I can merge this and release a new version.

File: src/mfs.js [hunk 1]
```diff
@@ -71,6 +71,10 @@ module.exports = function (options) {
 
     ipfsCtl.files.read(store.baseDir + opts.key, {}, (err, stream) => {
       if (err) {
+        if (err.toString().indexOf('does not exist') > -1) {
+          err.notFound = true
+        }
```
Comment (diasdavid) [inline src/mfs.js]: 👍

**Comment by diasdavid** [comment]
@hacdias go ahead and release a new version :)

**Pull request merged**